kratos/python_scripts/run_tests.py

```diff
@@ -297,6 +297,16 @@ def main():
             signalTime
         )
 
+    sys.stderr.flush()
+
+    # Run the internal tests (does the same as run_cpp_tests.py)
+    print('Running internal tests', file=sys.stderr)
+    try:
+        Tester.SetVerbosity(Tester.Verbosity.TESTS_OUTPUTS)
```

Comment:
I would use menos verbosity like test lists or even less

Reply:
FAILED_TESTS_OUTPUTS?

Reply:
I would go for PROGRESS which is similar to the python ones showing "." for each test and error messages for the failed.